lcm/property.go

```diff
@@ -46,15 +46,13 @@ type Property struct {
 type Properties []*Property
 
 //New returns a new properties
-func (ps Properties) New(Name string, Value interface{}) *Property {
+func (ps Properties) New(Name string, Value interface{}) *Properties {
```

Comment:
At first glance，this method name means create a new Properties. May use `AddProperty` instead?